boudicca.base/publisher-event-html/src/main/resources/templates/events/event.hbs

```diff
@@ -1,7 +1,7 @@
 <div class="event">
         <div class="event-image {{#if event.category}}event-image-{{event.category}}{{/if}}">
             {{#if event.pictureUuid}}
-                <img src="/picture?uuid={{event.pictureUuid}}" style="height:100px;max-height:250px;max-width:300px;" />
+                <img src="/picture?uuid={{event.pictureUuid}}" style="height:100px;max-height:250px;max-width:300px;" alt="{{event.name}} bild" aria-label="Event Bild" aria-labelledby="image" />
```

Comment:
are we sure that alt and aria-label should be different from each other?

Reply:
I was experimenting tags, we should only keep alt, since it's an `img` tag itself and represents it's role. but, with the current screen reader extension, I was unable to validate neither. non is read by the screen reader.